libraries/GROOT/GRootObjectManager.cxx

```diff
@@ -267,12 +267,12 @@ TH1 *GRootObjectManager::GetLast1D(TObject *object) {
 
 
 
-TH2    *GRootObjectManager::GetNext2D(TObject *object) { }
-TH2    *GRootObjectManager::GetLast2D(TObject *object) { }
+TH2    *GRootObjectManager::GetNext2D(TObject *object) { return 0; }
+TH2    *GRootObjectManager::GetLast2D(TObject *object) { return 0; }
```

Comment:
i'll start to get rid of some of the debuging statements too, which should
help a bit.

On Fri, May 1, 2015 at 9:41 AM, Ryan Dunlop <notifications@github.com>
wrote:

> In libraries/GROOT/GRootObjectManager.cxx
> <https://github.com/pcbend/GRSISort/pull/298#discussion_r29501746>:
>
> > @@ -267,12 +267,12 @@ TH1 *GRootObjectManager::GetLast1D(TObject *object) {
> >
> >
> >
> > -TH2    *GRootObjectManager::GetNext2D(TObject *object) { }
> > -TH2    *GRootObjectManager::GetLast2D(TObject *object) { }
> > +TH2    *GRootObjectManager::GetNext2D(TObject *object) { return 0; }
> > +TH2    *GRootObjectManager::GetLast2D(TObject *object) { return 0; }
>
> Yeah I'm just trying to make it so apple doesn't barf everywhere even
> during a successful compilation. It looks like that might take ROOT6 though
> because of the dictionary generation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pcbend/GRSISort/pull/298/files#r29501746>.
>